mantidproject/mantid
# Add images to release notes

**Description of work.**
Add some images to the release notes for 3.13

**To test:**
 - Check the docs for:
   - ui
   - indirect inelastic
   - diffraction

_No associated issue_

Does this update require release notes?


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
